crates/bevy_window/src/window.rs

```diff
@@ -128,8 +126,9 @@ impl NormalizedWindowRef {
 )]
 #[reflect(Component, Default)]
 pub struct Window {
-    /// The cursor of this window.
-    pub cursor: Cursor,
+    /// The cursor options of this window. Cursor icons are set with the `Cursor` component on the
+    /// window entity.
+    pub cursor_options: CursorOptions,
```

Comment:
Probably best tackled in follow-up, but I don't think this makes any sense to tie to the `Window` struct / entity.